CharlieBackend.Core/Entities/EmailData.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace CharlieBackend.Core.Entities
+{
+    public class EmailData
```

Comment:
This souldnt be located in entities since it's not in our schema. Place this into dto folder